packages/shared-state-wifi_links_info/Makefile

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (C) 2019 Gioacchino Mazzurco <gio@altermundi.net>
+#
+# This is free software, licensed under the GNU Affero General Public License v3.
+#
+
+include $(TOPDIR)/rules.mk
+
+GIT_COMMIT_DATE:=$(shell git log -n 1 --pretty=%ad --date=short . )
+GIT_COMMIT_TSTAMP:=$(shell git log -n 1 --pretty=%at . )
+
+PKG_NAME:=shared-state_links_info
+PKG_VERSION=$(GIT_COMMIT_DATE)-$(GIT_COMMIT_TSTAMP)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)
+	TITLE:=WIFI links information module for shared-state
+	CATEGORY:=LibreMesh
+	MAINTAINER:=
+	URL:=http://libremesh.org
+	DEPENDS:=+lua +luci-lib-jsonc +random-numgen \
+		+ubus-lime-utils +lime-system shared-state 
```

Comment:
It seems to me `ubus-lime-utils` isn't really used by this package, am I wrong?

Reply:
this dependency was added because the module uses node_status.lua from that package.

Reply:
ok